ote/src/cljc/ote/util/transport_operator_util.cljc

```diff
@@ -9,4 +9,9 @@
         filename (string/lower-case (str
                                       (string/replace operator-name " " "_")
                                       "_gtfs.zip"))]
-    filename))
\ No newline at end of file
+    filename))
+
+(def every-postal-code-regex #"(?i)^[a-z0-9][a-z0-9\- ]{0,9}$")
```

Comment:
"_Sallitaan syötteeksi jatkossa numeroiden lisäksi esim. kirjaimia ja ainakin merkki "-". Minimimerkkimäärä 1 ja maksimimerkkimääräksi esim 16._"
1) regex  allows: 1-10 chars. Just making sure, is 10 intentional or should it be 16?
2) regex allows strings like `a---------`. 
Without exact rules that's as good as any, but would following suit better? 
```suggestion
(def every-postal-code-regex #"(?i)^(?=.{1,10}$)[a-z0-9]+-?[a-z0-9]+$")
```
Allows total string length of 1-10 for a string consisting of: 
one or more a-z0-9
zero or one dash
one or moreaz0-9

Reply:
Does this allow empty spaces? NNN NNN ?

Reply:
It didn't but I fixed it.